### DISCUSSION TITLE
Why spaces are necessary

### DISCUSSION THREAD

**Discussion by littledu** [semantic-release/semantic-release#2225] [Commit Convention]
## Current behavior

**diff**

`feat: can work if have spaces between :`

`feat:can not work if no spaces between :`

Why spaces are necessary？

**Comment by travi** [comment]
the details of each commit convention are defined by the conventions themselves. semantic-release simply parses commits according to the chosen convention. you can find more details [in our docs](https://semantic-release.gitbook.io/semantic-release/#commit-message-format)
